homework/index.js

```diff
@@ -30,13 +30,118 @@
     return elem;
   }
 
+  function setInnerText(repository, nthChild, text) {
+    document.querySelector('tbody').childNodes[nthChild].lastChild.innerText = text;
+  }
+
+  function presentRepositories(leftHandColumn, repository) {
+    const table = createAndAppend('table', leftHandColumn);
+    const tBody = createAndAppend('tbody', table);
+    const headsOfDetails = ['Repository:', 'Description:', 'Forks:', 'Updated:'];
+    for (let i = 0; i < 4; i++) {
+      const tRow = createAndAppend('tr', tBody);
+      for (let j = 0; j < 2; j++) {
+        createAndAppend('td', tRow);
+        tRow.childNodes[0].innerText = headsOfDetails[i];
+      }
+    }
+    const tData = document.querySelector('tr').lastChild;
+    createAndAppend('a', tData, {
+      href: repository.html_url,
+      target: '_blank',
+      id: 'link-repo',
+    });
+    document.getElementById('link-repo').textContent = repository.name;
```

Comment:
I would suggest that you save the return value of `createAndAppend` above in a variable and assign `textContent` to it. This way you need not lookup for the element using the `document.getElementById` method and can save some execution time 🙂

Reply:
```
const repoName = createAndAppend('a', tData, {
  href: repository.html_url,
  target: '_blank',
  id: 'link-repo',
});
repoName.textContent = repository.name;
```

Reply:
Applied. Thanks, Yash :pray: